src/shared-components/dropdown/desktopDropdown.tsx

```diff
@@ -84,10 +86,10 @@ const DesktopDropdown = ({
         <DropdownOptionsContainer
           isOpen={isOpen}
           optionsContainerMaxHeight={optionsContainerMaxHeight}
-          role="listbox"
+          role="menu"
           aria-activedescendant={value}
-          tabIndex={isOpen ? 0 : -1}
           aria-labelledby="select-input-box"
+          aria-hidden={!isOpen}
```

Comment:
A `listbox` or `menu` must have focusable content to be valid, so we hide this from the accessibility tree when the menu is not open.